intercom/README.md

```diff
@@ -0,0 +1,84 @@
+# Intercom
+
+## Overview
+[Intercom][1] is a customer communication platform that enables businesses to engage with their users through in-app messaging, email, and chat. It offers features like live chat, automated messaging, and customer support tools, making it easier for companies to provide personalized customer experiences.
+
+The Intercom integration seamlessly collects admin activities, data events, conversations, news items, and ticket data, and ingests them into Datadog for comprehensive analysis using webhooks.
+
+## Setup
+
+Follow the instructions below to configure this integration for your Intercom account.
+
+### Configuration
+
+#### Webhook Configuration
+Configure the Datadog endpoint to forward Intercom events as logs to Datadog. See [Intercom webhook overview][3] for more details.
+
+- #### Webhook URL generation
+    - Select an existing API key or create a new one by clicking one of the buttons below:<!-- UI Component to be added by DataDog team -->
+- #### Register a new application
+    1. Sign in to your [Intercom][2] account using a user account with full access to Apps and Integrations.
+    2. Go to **Settings**.
+    3. In the Integrations section, select **Developer Hub**.
+    4. Click **New app**.
+    5. Fill in the required details for your application, such as the name and associated workspace.
+    6. Click **Create app**.
+- #### Configure webhook topics
+    1. Select Required permissions
+        1. After creating the app, navigate to the *Authentication* section using the left-side menu and click **Edit** button in the upper-right corner.
```

Comment:
`After creating the app, go to the Authentication section in the left-side menu, and click the Edit button in the top-right corner.`

Reply:
Done changes as per suggestions.